### DISCUSSION TITLE
Why EfficienAD become less effective after update the version?

### DISCUSSION THREAD

**Discussion by lzes** [open-edge-platform/anomalib#1783] [Q&A]
This is the result now
![000](https://github.com/openvinotoolkit/anomalib/assets/102123656/7adfea5f-895d-468d-878f-4e78f6da33c6)
here is the result used the version about one month ago
![000](https://github.com/openvinotoolkit/anomalib/assets/102123656/7d00f9a1-3610-4f8f-8745-4b373bba8ead)
I only find that the batchsize changed from 1 to 32,is this the reason?

**Comment by abc-125** [comment]
Hello, yes, this is most probably the reason. The paper uses batch size 1.

**Reply by blaz-r** [reply]
I also checked the code and this is now part of EfficientAD:
https://github.com/openvinotoolkit/anomalib/blob/9482d34049dad874fcf68edbc79454638700813a/src/anomalib/models/image/efficient_ad/lightning_model.py#L311-L319

Isn't efficientAD supposed to work without imagenet normalization?

**Reply by abc-125** [reply]
> Isn't efficientAD supposed to work without imagenet normalization?

Yes, there is an internal normalization. This transformation should have only `Resize`.
